apps/web-connect/src/features/hooks/contract/useMintBatch.ts

```diff
@@ -2,8 +2,11 @@ import { useState } from "react";
 import { useAccount, useWriteContract } from "wagmi";
 import { NodeLicenseAbi, config } from "@sentry/core";
 import { CURRENCIES, Currency } from "../shared";
+import { errorNotification, successNotification } from "@/features/checkout/components/notifications/NotificationsComponent";
+import { getPriceForQuantity as getPriceForQuantityCore } from "@sentry/core";
+import { convertEthAmountToXaiAmount } from "@/utils/convertEthAmountToXaiAmount";
 
-export const MAX_BATCH_SIZE = 175;
+export const MAX_BATCH_SIZE = 2;
```

Comment:
This will be reverted once the PR is moved from Draft, left here for testing in PR env.